charts/ai-software-templates/chatbot/templates/deployment-model-server.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
```

Comment:
I know the model is currently coupled with the app over in ai-lab-tempates but Maysun did start providing model only deployments and it makes me wonder if we should split out the model from these existing apps into a separate chart off of ai-sofwtare-templates/vllm-model-server-blah-blah and then just have the customer run the model chart before the app chart.

That said, I acknowledge I may not understand the dirty details of your conversion work, and that is too much of a change to take on.

WDYT @thepetk  .... does it make things really complicated to move the ai model chatbot is using into a separate chart?

Reply:
I think is aligned with my idea of adding the [INCLUDE_FILES](https://github.com/thepetk/ai-lab-helm-charts/blob/feat/automatic-template-conversion/scripts/envs/chatbot-ai-sample#L12-L20) env var. I had this case exactly in my mind looking to the future direction of the templates.

So in case of no-model-template I was envisioning indeed that we will generate a new helm chart importing only the deployment related resources.

Reply:
OK as long as it is on your radar and you'll address it in a subsequent Jira or f/up PR for this Jira I'm good here.

Just clarify for me if it is this Jira or a subsequent one and we can mark this resolved @thepetk

Reply:
I would say is part of RHDHPAI-182. For example in a case of a model-only chart we will create a customized "env" file which will set a different list of resources.

Reply:
OK can we tweak the description of RHDHPAI-182 then so that in conjunction with the model only helm chart, we add options to the app helm charts like chatbot so that they only provision the application, and defer provisioning of the model to the model only helm chart.

Reply:
After a second thought (I had in mind only the "model only" case) I feel the "model-only" case will be covered from RHDHPAI-182, and the "app-only" case as you mention earlier it should be covered as part of RHDHPAI-379 (e.g the final issue for the conversion). More or less everything is in order with this PR, I would like just to try all the combinations and provide more insights for the documentation.

Reply:
sounds good .... I added this distinction of model-only vs. app-only vs. app+model into the acceptance criteria for RHDHPAI-379